custom/icds_reports/tasks.py

```diff
@@ -48,8 +49,12 @@ def move_ucr_data_into_aggregation_tables(date=None, intervals=2):
 
     monthly_dates.append(date)
 
-    if hasattr(settings, "ICDS_UCR_DATABASE_ALIAS") and settings.ICDS_UCR_DATABASE_ALIAS:
-        with connections[settings.ICDS_UCR_DATABASE_ALIAS].cursor() as cursor:
+    try:
+        db_alias = connection_manager.get_django_db_alias(ICDS_UCR_ENGINE_ID)
+    except KeyError:
+        pass
```

Comment:
would there be any value in extracting this out into a function? saw it in a few places

Reply:
6adac74